.travis.yml

```diff
@@ -6,8 +6,6 @@ android:
   licenses:
   - android-sdk-license-.+
 before_install:
```

Comment:
Essas alterações são necessárias para o funcionamento dessa release ? Caso não, sugira retirá-las desse PR e criar um focado somente nelas.

Reply:
Sim, são necessárias para que PR externos ao repositório passem no Travis. Há um PR somente para o Travis em que essas modificações são realizadas: [Try to fix travis error](https://github.com/Escola-em-Casa/android-escola-em-casa/pull/54)